Firestore/Example/Tests/Util/FSTEventAccumulator.h

```diff
@@ -23,7 +23,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-typedef void (^FSTGenericEventHandler)(id _Nullable, NSError *error);
+typedef void (^FSTValueEventHandler)(id _Nullable, NSError *error);
```

Comment:
Isn't `error` nullable too?

Reply:
Fixed here and below. Used _Nullable as in the public header file.

Reply:
Hm, not sure what happened here. Fixed.